src/commonMain/kotlin/character/Class.kt

```diff
@@ -74,7 +74,7 @@ abstract class Class(
     abstract fun talentFromString(name: String, ranks: Int): Talent?
 
     // Class resource
-    abstract val resourceType: Resource.Type
+    abstract val resourceType: List<Resource.Type>
```

Comment:
Minor, as this is now a `List`, a better name might be `resourceTypes`.

Reply:
Changed everywhere.